datahub-actions/build.gradle

```diff
@@ -148,7 +148,19 @@ docker {
     "slim": [suffix: "-slim", args: [APP_ENV: "slim", RELEASE_VERSION: python_docker_version]],
     "full": [suffix: "", args: [APP_ENV: "full", RELEASE_VERSION: python_docker_version]]
   ]
+  // This task is intended to build the slim image
+  //target 'ingestion-base-slim' //Review if this needs to be handled by bake
```

Comment:
This stuff should be handled by bake ideally

Reply:
oh, yeah i just copied this from another file. i'll remove it